### PR TITLE
Add lemmy documentation

### PR DESCRIPTION
**Pull request by jahway603**
Add lemmy documentation

### CODE REVIEW

File: pages/04.applications/10.docs/lemmy/' [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
```
Comment (tituspijean) [inline pages/04.applications/10.docs/lemmy/':1]: To be deleted 😉

**Review by tituspijean — CHANGES_REQUESTED** [review]
Thank you! One small oversight to be fixed.

File: pages/04.applications/10.docs/lemmy/' (hunk 1 above)
Comment (jahway603) [reply pages/04.applications/10.docs/lemmy/':1]: Missed that, thanks

File: pages/04.applications/10.docs/abantecart/app_abantecart.md
Comment (tituspijean) [inline pages/04.applications/10.docs/abantecart/app_abantecart.md:1]: Same thing. Please rebase against master and force-push ;)

Comment (jahway603) [reply pages/04.applications/10.docs/abantecart/app_abantecart.md:1]: Unsure of how to do that - any suggestions?

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**